test/config/install_default.json

```diff
@@ -1,31 +1,32 @@
-      "template": "ora-ubuntu-1604.ova",
-      "ports": {
-        "_comment": "These are the northbound rest API port assignments",
-        "http": 8080,
-        "https": 443
-      },
-      "install": {
-        "rackhd":       {"repo":"https://github.com/rackhd/rackhd","branch":"master"},
-        "on-core":      {"repo":"https://github.com/rackhd/on-core","branch":"master"},
-        "on-dhcp-proxy":{"repo":"https://github.com/rackhd/on-dhcp-proxy","branch":"master"},
-        "on-http":      {"repo":"https://github.com/rackhd/on-http","branch":"master"},
-        "on-statsd":    {"repo":"https://github.com/rackhd/on-statsd","branch":"master"},
-        "on-syslog":    {"repo":"https://github.com/rackhd/on-syslog","branch":"master"},
-        "on-tasks":     {"repo":"https://github.com/rackhd/on-tasks","branch":"master"},
-        "on-taskgraph": {"repo":"https://github.com/rackhd/on-taskgraph","branch":"master"},
-        "on-tftp":      {"repo":"https://github.com/rackhd/on-tftp","branch":"master"},
-        "on-tools":     {"repo":"https://github.com/rackhd/on-tools","branch":"master"},
-        "on-imagebuilder": {"repo":"https://github.com/rackhd/on-imagebuilder","branch":"master"},
-        "on-wss":       {"repo":"https://github.com/rackhd/on-wss","branch":"master"}
-        },
-      "skupack": [
-        "https://github.com/RackHD/on-skupack"
-        ],
-      "proxy": {
-        "host": "",
-        "port": ""
-      }
-  }
-}
+{
+  "install-config": {
+      "template": "ora-ubuntu-1604.ova",
+      "ports": {
+        "_comment": "These are the northbound rest API port assignments",
+        "http": 8080,
+        "https": 443,
+        "amqp":5672
+      },
+      "install": {
+        "rackhd":       {"repo":"https://github.com/rackhd/rackhd","branch":"master"},
+        "on-core":      {"repo":"https://github.com/rackhd/on-core","branch":"master"},
+        "on-dhcp-proxy":{"repo":"https://github.com/rackhd/on-dhcp-proxy","branch":"master"},
+        "on-http":      {"repo":"https://github.com/rackhd/on-http","branch":"master"},
+        "on-statsd":    {"repo":"https://github.com/rackhd/on-statsd","branch":"master"},
+        "on-syslog":    {"repo":"https://github.com/rackhd/on-syslog","branch":"master"},
+        "on-tasks":     {"repo":"https://github.com/rackhd/on-tasks","branch":"master"},
+        "on-taskgraph": {"repo":"https://github.com/rackhd/on-taskgraph","branch":"master"},
+        "on-tftp":      {"repo":"https://github.com/rackhd/on-tftp","branch":"master"},
+        "on-tools":     {"repo":"https://github.com/rackhd/on-tools","branch":"master"},
+        "on-imagebuilder": {"repo":"https://github.com/rackhd/on-imagebuilder","branch":"master"},
+        "on-wss":       {"repo":"https://github.com/rackhd/on-wss","branch":"master"}
+        },
+      "skupack": [
+        "https://github.com/RackHD/on-skupack"
+        ],
+      "proxy": {
+        "host": "",
+        "port": ""
+      }
+  }
+}

```

Comment:
Not a huge deal, but there is a white-space change on all lines in this file.  The carriage return (\r) has been added to the new line (\n) at the end of each line.  Most likely based on the editor that was used.
Just noting it in case others are wondering why all of the lines show up in the diff.